### PR TITLE
BEP4: 256-bit Tokens

### PR DESCRIPTION
**Pull request by wjmelements**
#### Changes
This adds a proposal to increase the number of bits for tokens from 63 to 256.
Reviewers @notatestuser @huangsuyu

### CODE REVIEW

**Comment by darren-liu** [comment]
Even though there exist many

File: BEP4.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+This BEP is a WIP. 
+
+## 4.  Motivation
+
+The int64 type used in BEP2 is insufficient to handle existing use cases.
+The built-in base unit decimalization is 1e8, or 100,000,000.
+This allows slightly more then 92 billion tokens, though the protocol currently limits total supply to just 90 billion.
+However, several top cryptocurrencies already have supplies exceeding that limit.
+
+* 119 billion Dogecoin
```
Comment (lacvinhky) [inline BEP4.md:36]: Ọ

**Review by Jonsms1 — APPROVED** [review]

**Review by Jonsms1 — APPROVED** [review]

**Comment by nguyenducsang** [comment]
Ok

File: BEP4.md (hunk 1 above)
Comment (nguyenducsang) [reply BEP4.md:36]: Ok

**Review by nguyenducsang — CHANGES_REQUESTED** [review]
Hâ

**Comment by trastvolt** [comment]
wjmelements:bep-change-token-units

**Comment by brilliant-lx** [comment]
outdate, archive it.

**Pull request closed without merging**